pkg/ccl/changefeedccl/sink_kafka.go

```diff
@@ -605,6 +605,15 @@ func (s *kafkaSink) handleBufferedRetries(msgs []*sarama.ProducerMessage, retryE
 		// SendMessages will attempt to send all messages into an AsyncProducer with
 		// the client's config and then block until the results come in.
 		lastSendErr = newProducer.SendMessages(msgs)
+		if lastSendErr != nil {
+			// nolint:errcmp
+			if sendErrs, ok := lastSendErr.(sarama.ProducerErrors); ok && len(sendErrs) > 0 {
```

Comment:
errors.As() doesn't work?

Reply:
That didn't pass the linter either 🤷 .  There are some other locations in the codebase that do a cast where either `nolint` is used or the file is ignored in `build/bazelutil/nogo_config.json`.